src/VisualStudio/Core/Test/DebuggerIntelliSense/CSharpDebuggerIntellisenseTests.vb

```diff
@@ -459,6 +459,162 @@ Namespace Microsoft.VisualStudio.LanguageServices.UnitTests.DebuggerIntelliSense
             End Using
         End Sub
 
+        <WorkItem(1124544)>
+        <ConditionalFact(GetType(x86)), Trait(Traits.Feature, Traits.Features.DebuggingIntelliSense)>
+        Public Sub CompletionUsesContextBufferPositions()
+            Dim text = <Workspace>
+                           <Project Language="C#" CommonReferences="true">
+                               <Document>
+                                   e.InnerException
+{"Exception of type 'System.Exception' was thrown."}
```

Comment:
This could be any text--it's just padding to make the "immediate window" look like there's a lot of scrollback. This is not output that the test will examine.